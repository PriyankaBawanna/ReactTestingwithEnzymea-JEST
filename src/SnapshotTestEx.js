import React from "react";

export default function SnapshotTestEx() {
  return (
    <>
      <h1> Snap Shot Test </h1>
      <h2> Snap Shot Test</h2>
      <h3> Snap Shot Test</h3>
      <h4> Snap Shot Test</h4>
      <h5> Snap Shot Test</h5>
      <h6> Snap Shot Test</h6>
    </>
  );
}
