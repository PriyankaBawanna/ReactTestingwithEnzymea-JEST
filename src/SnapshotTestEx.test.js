import { shallow } from "enzyme";
import SnapshotTestEx from "./SnapshotTestEx";

test("snapShot Testing ", () => {
  const wrapper = shallow(<SnapshotTestEx />);
  expect(wrapper).toMatchSnapshot();
});
