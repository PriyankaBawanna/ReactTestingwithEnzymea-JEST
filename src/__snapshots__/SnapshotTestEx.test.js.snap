// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapShot Testing  1`] = `
<Fragment>
  <h1>
     Snap Shot Test 
  </h1>
  <h2>
     Snap Shot Test
  </h2>
  <h3>
     Snap Shot Test
  </h3>
  <h4>
     Snap Shot Test
  </h4>
  <h5>
     Snap Shot Test
  </h5>
  <h6>
     Snap Shot Test
  </h6>
</Fragment>
`;
